modules/react-components/src/components/list/resource-list/resource-list-item.tsx

```diff
@@ -126,13 +126,14 @@ export const ResourceListItem: FunctionComponent<ResourceListItemPropsInterface>
     return (
         <List.Item className={ classes } data-testid={ testId } { ...rest }>
             <Grid>
-                <Grid.Row columns={
-                    metaContent instanceof Array
-                        ? (metaContent.length + 2) as StrictGridRowProps[ "columns" ]
-                        : (itemHeader || itemDescription)
-                            ? 3
-                            : 2
-                }>
+                <Grid.Row
+                    columns={
+                        metaContent instanceof Array
+                            ? (metaContent.length + 2) as StrictGridRowProps[ "columns" ]
+                            : (itemHeader || itemDescription)
+                                ? 3
+                                : 2
+                    }>
```

Comment:
Shall we bring `>` to next line?

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/2712/commits/822ffa6c9b9d9f29aad197efcd574ef27c3e7440 & https://github.com/wso2/identity-apps/pull/2712/commits/f04b53ae80f3c688374854052c552a5f698db22f.